src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -132,6 +132,7 @@
     <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="2.18.0.1400" />
     <PackageReference Include="AvalonEdit" Version="6.3.0.90" CopyXML="true" />
     <PackageReference Include="Greg" Version="2.5.0.5076" />
+    <PackageReference Include="LiveChartsCore.SkiaSharpView.WPF" Version="2.0.0-beta.855" />
```

Comment:
why does DynamoCoreWPF need to reference livecharts? IE why does the core need to depend on a node library? Doesn't that sound backward?

Reply:
removed